tests/e2e/testsLibrary/LsTests.ts

```diff
@@ -68,8 +68,20 @@ const ide: Ide = e2eContainer.get(CLASSES.Ide);
 
  export function codeNavigation(openedFile: string, line: number, char: number, codeNavigationClassName: string) {
     test('Codenavigation', async () => {
-        await editor.moveCursorToLineAndChar(openedFile, line, char);
-        await editor.performKeyCombination(openedFile, Key.chord(Key.CONTROL, Key.F12));
-        await editor.waitEditorAvailable(codeNavigationClassName);
+        try {
+            await editor.moveCursorToLineAndChar(openedFile, line, char);
+            await editor.performKeyCombination(openedFile, Key.chord(Key.CONTROL, Key.F12));
+            await editor.waitEditorAvailable(codeNavigationClassName);
+        } catch (err) {
+            if (err instanceof error.TimeoutError) {
+                Logger.warn('Code navigatino didn\'t work. Trying again.');
```

Comment:
typo `navigatino`

Reply:
It would also be helpful to have a comment with link to issue which describes the flakiness we are working around.

Reply:
I'll create an issue add it to the comment.

Reply:
link added